clients/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
Why do we need this?

Reply:
to be able to mock final classes with mockito

Reply:
This is not worth it: https://github.com/mockito/mockito/issues/1142

As I said, we can remove `final` from internal classes if needed for testing.

Reply:
good point, I removed. just wanted to avoid prod code change just for tests